atd-etl/app/process/helpers_import.py

```diff
@@ -564,6 +564,10 @@ def insert_crash_change_template(new_record_dict, differences, crash_id):
         if isinstance(new_record_dict[key], datetime.time):
             new_record_dict[key] = new_record_dict[key].strftime("%H:%M:%S")
 
+        # CRIS or the upstream ETL is representing this datum as a float, so cast it back
+        if key == "rpt_sec_speed_limit": 
+            new_record_dict[key] = int(new_record_dict[key])
```

Comment:
This is the fix for the literal, actual error that is found in the prefect channel this morning.